PizzaBuilder/PizzaBuilder/Program.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PizzaBuilder
+{
+    public interface IPizzaBuilder
+    {
+       public void BuildDough();
+       public void BuildSauce();
+       public void BuildTopping();
+       public void reset();
+    }
+
+    public class WhitePizzaBuilder : IPizzaBuilder
+    {
+        private whitePizza WhitePizza = new whitePizza();
+
+        public WhitePizzaBuilder()
+        {
+            this.reset();
+        }
+        public void BuildDough()
+        {
+            this.WhitePizza.Add("White Dough");
+        }
+
+        public void BuildSauce()
+        {
+            this.WhitePizza.Add("Sauce");
+        }
+
+        public void BuildTopping()
```

Comment:
It would be nice if you specified some toppings like (Broccoli, Mushroom, onions) or what you want rather than just writing the word topping.